src/utils/misc.ts

```diff
@@ -1,8 +1,8 @@
 import { ethers } from 'ethers';
 import { keyBy, merge, values, orderBy, toNumber, isString } from 'lodash';
 
-import { createToken } from '@src/core/store/modules/tokens/tokens.selectors';
-import { Balance, Token, TokenView } from '@src/core/types';
+import { createToken } from '@store/modules/tokens/tokens.selectors';
```

Comment:
I think `createToken` probably should not have been placed inside selectors, since its more of a helper function. That is why its not reexported anywhere in the module. Lets figure it out in a refactor PR later.